### ISSUE TITLE
[MSX2] Golvellius 2 shows green stripes while scrolling on top view angle. 

### ISSUE THREAD

**Issue by SegaSnatcher** [MiSTer-devel/MSX1_MiSTer#22]
I'm no MSX2 expert, so I apologize if this is true to real hardware behavior, but thought I'd bring it up just in case it is a bug. 

https://www.youtube.com/watch?v=-VR6PvMgnLA

**Comment by vanfanel** [comment]
This seems to be a real bug: the game is supposed to scroll.

https://www.youtube.com/watch?v=6ABtz7Zk6rE

**Comment by GuerreroNinja** [comment]
In emulators:

If you use a MSX2 machine in FMSX or BlueMSX retroarch cores, you get the green bars. If you configure the cores to use a MSX2+ config, the scroll shows ok.

In real machine:

I could not test the game on a F1-XDmk2 (Japanese MSX2 machine) as it hangs when selecting new game. I tried 2 DSK versions, both the same result. Weird, I will test some more later, and I will test also on two MSX2+ machines.

**Comment by MP2E** [comment]
Pretty sure the green bars are normal behavior when you are using an MSX2 VDP

If it detects an MSX2+/TR VDP, it scrolls normally without the green bars

**Comment by MP2E** [comment]
https://youtu.be/4R779hMGGC4?t=906

This is on a real MSX2, looks like the green bars are in fact normal for MSX2 computers

**Comment by GuerreroNinja** [comment]
Tested on 2 real MSX2+ machines, scroll works as it should, no green bars.

However, when I tried to test it on a MSX2 machine, the game hangs. Does not matter if I continue a game or I create a new one, it hangs. I think this is something related to MegaFlashRom disk handling. I tried with SofaRunIt, same behaviour. The machine does not have a working FDD nor a GOTEK so I can't test it with a real disk image right now.

I think the green bars may be the normal behaviour on MSX2 machines, but the one I have now the game hangs.
